lib/Search/CardSearchResultEntry.php

```diff
@@ -40,7 +40,7 @@ public function __construct(Board $board, Stack $stack, Card $card, $urlGenerato
 			),
 			$card->getTitle(),
 			$board->getTitle() . ' » ' . $stack->getTitle(),
-			$urlGenerator->linkToRouteAbsolute('deck.page.index') . '#/board/' . $board->getId() . '/card/' . $card->getId(),
+			$urlGenerator->linkToRouteAbsolute('deck.page.index') . 'card/' . $card->getId(),
```

Comment:
We should use the actual route directly:

Given this one in routes.php
```
		['name' => 'page#redirectToCard', 'url' => '/card/{cardId}', 'verb' => 'GET'],
```

To have runtime validation that the route also exists:
```suggestion
			$urlGenerator->linkToRouteAbsolute('deck.page.redirectToCard', [ 'cardId' => $cardId ]),
```